src/System.Diagnostics.Process/tests/ProcessTests.cs

```diff
@@ -1804,6 +1804,41 @@ public void TestLongProcessIsWorking()
             Assert.True(p.HasExited);
         }
 
+        [Fact]
+        public void GetProcesses_LongProcessName()
+        {
+            string commandName = "sleep";
+
+            // sleep program doesn't exist on some flavor
```

Comment:
Where does / doesn't it exist?

Reply:
It was a Dan suggestion https://github.com/dotnet/corefx/issues/34437#issuecomment-460097674 /cc @danmosemsft

Reply:
sleep is part of GNU coreutils. quite common on modern UNIX systems that support bash. However there maybe a different set of params for sleep on OSX.